scapy/contrib/vqp.py

```diff
@@ -52,13 +52,13 @@ class VQPEntry(Packet):
             3080: "ResMACAddress"
         }),
         FieldLenField("len", None),
-        ConditionalField(IPField("datatom", "0.0.0.0"),
-                         lambda p: p.datatype == 3073),
-        ConditionalField(MACField("MAC_data", "00:00:00:00:00:00"),
-                         lambda p: p.datatype == 3078 or p.datatype == 3080),
-        ConditionalField(StrLenField("data", None,
-                                     length_from=lambda p: p.len),
-                         lambda p: p.datatype not in [3073, 3078, 3080]),
+        MultipleTypeField([
+            (IPField("datatom", "0.0.0.0"), lambda p: p.datatype == 3073),
```

Comment:
I wasn't sure if datatom should be changed to data, but I included it since the field should default if datatype is neither for datatom or data.